stream.go

```diff
@@ -89,6 +89,15 @@ func (s *Stream) Metadata() *Dictionary {
 	return newDictionaryFromC(s.c.metadata)
 }
 
+// https://ffmpeg.org/doxygen/7.0/structAVStream.html
```

Comment:
Could you correct the link to `https://ffmpeg.org/doxygen/7.0/structAVStream.html#a50d250a128a3da9ce3d135e84213fb82` instead?

And correct the link of `Metadata()` as well just above?